packages/dev/core/src/Particles/baseParticleSystem.ts

```diff
@@ -406,6 +454,33 @@ export class BaseParticleSystem implements IClipPlanesHolder {
         );
     }
 
+    protected _setEngineBasedOnBlendMode = (blendMode: number) => {
+        let mode;
+        switch (blendMode) {

```

Comment:
NIT: you can simply use only blendMode. In the switch instead of `mode =`, use `blendMode =` so it is never undefined and simplifies the last call to `this._engine.setAlphaMode(mode);` I guess ?